infra/Pos/Communication/BiP.hs

```diff
@@ -33,7 +34,9 @@ bipPacking = Packing
     }
 
 biPackMsg :: Bi.Encoding -> LBS.ByteString
-biPackMsg = Bi.toLazyByteString
+biPackMsg = Builder.toLazyByteStringWith strategy mempty . Bi.toBuilder
+  where
+    strategy = Builder.untrimmedStrategy 1024 4096
```

Comment:
This is used for everything that we send via `ConversationActions`. There's probably no one-size-fits-all choice. We could change the interface so that the caller can choose.